ci/pipeline.yml

```diff
@@ -1620,3 +1635,11 @@ resource_types:
     repository: time-resource
     aws_region: us-gov-west-1
     tag: latest
+
+- name: secrets
```

Comment:
As a next step, would love to migrate these to Credhub: https://github.com/cloud-gov/internal-docs/blob/main/docs/runbooks/Concourse/credhub-credentials.md#migrating-to-credhub